module/os/linux/zfs/zfs_vfsops.c

```diff
@@ -25,6 +25,7 @@
 
 /* Portions Copyright 2010 Robert Milkowski */
 
+#include <sys/zfs_context.h>
```

Comment:
Since this will just be compiled in kernel space I believe you could switch this include to `sys/timer.h` to avoid pulling in a bunch of additional headers.